Automattic/jetpack
# Jetpack Licenses: Add links to license activation screen

Fixes 1201509629272450-as-1201532056430595

#### Changes proposed in this Pull Request:


This adds some helpful product-specific links to the screen shown after a license is successfully activated.

<img width="1098" alt="image" src="https://user-images.githubusercontent.com/42627630/147158834-d95e5307-d405-4059-987b-070fa6fe05a6.png">

#### Jetpack product discussion


p1HpG7-e3y-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






For each of the Jetpack products and plans do the following:

1. On a new site, visit `/wp-admin/admin.php?page=jetpack#/license/activation` and activate a license.
2. Verify that the success page shows with an appropriate external link, and a button that goes to the `my-plan` page.
